### ISSUE TITLE
Delaytest should use a single looping audio track

### ISSUE THREAD

**Issue by ghost** [atomic-software/dj#1]
Preferably at a higher hit rate than once per second, because I'm fairly certain it's going to be easier and much more natural for someone to tap to an actual beat as opposed to concentrating hard on trying to get good timing, then failing to do so because of that. Played sounds can also have an unpredictable, incalculable delay without low-level audio control. I'm gonna see if I can cobble together a solution.

**Comment by qoh** [comment]
I agree, though I think a bigger issue right now is that the game doesn't do _anything_ with the calculated delay value ;)
